flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumerBaseTest.java

```diff
@@ -756,6 +756,11 @@ public boolean isClearCalled() {
 		public void update(List<T> values) throws Exception {
 			clear();
 
+			addAll(values);
+		}
+
+		@Override
+		public void addAll(List<T> values) throws Exception {
 			if (values != null && !values.isEmpty()) {
```

Comment:
I think the check for `!values.isEmpty()` is not required and just boilerplate - the following line also works for empty lists.